dimod/generators/anti_crossing.py

```diff
@@ -84,6 +90,18 @@ def anti_crossing_loops(num_variables: int) -> BinaryQuadraticModel:
         quantum annealing of a 16-qubit problem. Nat Commun 4, 1903 (2013).
         https://doi.org/10.1038/ncomms2920
 
+    Examples:
+        The graphic lines of this example require the `Matplotlib <https://matplotlib.org>`_
```

Comment:
again, I don't think this is a very useful example without the visual. Better to have nothing.

Reply:
I think for these more advanced generators, we don't really get a lot of benefit from these examples. Anyway who knows what they are won't need them and other folks won't get much from a short example. Especially one without the visual. I would recommend adding "frustrated loop" to the SDK glossary and put images there and just link from here.

Reply:
Can do. Of course the visual will need to be in a SDK release before a link to it can be added ... :-)

Reply:
That's fine I think